test/utils.jl

```diff
@@ -18,6 +18,9 @@ function setup_core_worker(body)
     try
         body()
     finally
+        # let object ref finalizers run...
+        GC.gc()
+        yield()
```

Comment:
Is this required now? What happens if we don't run this?

Reply:
seg faults!  there's a race condition where the `shutdown_driver` may run before the finalizer tasks run (or even while they are running), causing hte `RemoveLocalReference` segfault when (somewhere in the stack) it refers to memory that's been cleaned up

Reply:
Can you add a comment here about this?

Reply:
ah crap I pushed the commit to the wrong branch.  it'll make it in w/ #138 I promise :)